flink-connectors/flink-hbase/src/main/java/org/apache/flink/addons/hbase/HBaseWriteOptions.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.addons.hbase;
+
+import java.io.Serializable;
+import java.util.Objects;
+
+import static org.apache.flink.util.Preconditions.checkArgument;
+
+/**
+ * Options for HBase writing.
+ */
+public class HBaseWriteOptions implements Serializable {
+
+	private static final long serialVersionUID = 1L;
+	private static final long DEFAULT_FLUSH_MAX_SIZE_BYTES = 2 * 1024 * 1024; // 2mb
```

Comment:
Suggest to use the `ConnectionConfiguration.WRITE_BUFFER_SIZE_DEFAULT` of hbase as the default (which is also 2M). Or is there any special reason for us to maintain a Flink-specified default write buffer size?

Reply:
Thanks for the tip.